src/metaMesh.cxx

```diff
@@ -142,10 +142,10 @@ MetaMesh::
 ~MetaMesh()
 {
   Clear();
-  for(unsigned int i=0;i<MET_NUM_CELL_TYPES;i++)
+  for(auto & i : m_CellListArray)
     {
-    delete m_CellListArray[i];
```

Comment:
@aylward It would be good if you double-checked the loop iteration changes.

The rest looks good.

Reply:
I've not seen this syntax before, but after digging a bit into it, it seems valid.  Cool!